pkgs/development/libraries/capstone/default.nix

```diff
@@ -3,25 +3,37 @@
 , cmake
 , fetchFromGitHub
 , fixDarwinDylibNames
+, static ? stdenv.hostPlatform.isStatic
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "capstone";
-  version = "4.0.2";
+  version = "5.0.1";
 
   src = fetchFromGitHub {
     owner = "aquynh";
```

Comment:
the repo owner has been renamed to `capstone-engine`, we might want to update that